### PR TITLE
Add team list-memberships command

### PR DESCRIPTION
**Pull request by patrickxb**
This doesn't do everything listed in 

https://keybase.io/ccsandbox/team-internal-mvp

but does a basic membership list (and has json output).

cc @malgorithms

### CODE REVIEW

File: go/protocol/keybase1/teams.go [hunk 1]
```diff
@@ -48,6 +48,13 @@ type PerTeamKey struct {
 	EncKID KID   `codec:"encKID" json:"encKID"`
 }
 
+type TeamMembers struct {
+	Owners  []string `codec:"owners" json:"owners"`
+	Admins  []string `codec:"admins" json:"admins"`
+	Writers []string `codec:"writers" json:"writers"`
+	Readers []string `codec:"readers" json:"readers"`
+}
```
Comment (maxtaco) [inline go/protocol/keybase1/teams.go]: does this conflict with what miles was doing?

File: protocol/avdl/keybase1/teams.avdl [hunk 2]
```diff
@@ -18,6 +18,14 @@ protocol teams {
       KID encKID;
   }
 
+  record TeamMembers {
+    array<string> owners;
+    array<string> admins;
+    array<string> writers;
+    array<string> readers;
```
Comment (maxtaco) [inline protocol/avdl/keybase1/teams.avdl]: Does this conflict with what miles is working on? There's a ticket for this I believe.  Also, even though most members show in lists of the form `"bob"`, there are people like `"bob%3"` which means bob after a reset at sigchain link 3.  By default, it's assumed `"bob%1"`, and you can leave off the `%1`.  We might want a special type for this.

Comment (patrickxb) [inline protocol/avdl/keybase1/teams.avdl]: This is just for display to the user, and based on the doc here:

https://keybase.io/ccsandbox/team-internal-mvp

**Comment by maxtaco** [comment]
There's also [CORE-5196](https://keybase.atlassian.net/browse/CORE-5196)

File: protocol/avdl/keybase1/teams.avdl (hunk 2 above)
Comment (patrickxb) [inline protocol/avdl/keybase1/teams.avdl]: Miles is working on 4879 which doesn't seem related
